grains/osdisk.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+import os  
+  
+def get_osdisk_stats():
+"""
+	Calculates and resturns the disk used, available and total capacity
+	in gigabytes. Calculations code from:
+	http://www.stealthcopter.com/blog/2009/09/python-diskspace/
+"""
```

Comment:
Done. I fixed indentation and trailing spaces as well, I wrote that in the server's vim and became a mess :)